docs/_layouts/default.html

```diff
@@ -55,6 +55,17 @@
               <li><a href="{{ '/debugging.html' | relative_url }}">Debugging common Bolt errors</a></li>
             </ul>
           </li>
+          <li>
+            Bolt Developer Updates
+            <ul>
+              {% for post in site.posts %}
```

Comment:
```suggestion
              {% for post in site.posts limit: 3 %}
```

We should limit the number of posts to something reasonable, otherwise the sidebar gets cluttered very quickly.

Reply:
I was thinking about this. I expect we'll quickly need some way to access the full list of posts, but limiting it for now seems fine.